### PR TITLE
changed title of event page

### PR DESCRIPTION
**Pull request by samyakshah3008**
### Issue

Closes issue ticket #623 

### What is the change?

Previously the event page title was `upcoming events` but events which we are displaying are already happened. So changed the title to `Recap of past events` 

### Is Development Tested?

- [X] Yes
- [ ] No

### Before Change Screenshot: 

image

### After Change Screenshot: 

image

For visual or interaction changes. Please provide a video / screenshot.

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -145,7 +145,7 @@
         />
       </a>
       <section class="event-details">
-        <h2 class="event-upcoming">Upcoming Events</h2>
+        <h2 class="event-upcoming">Recap of our past events</h2>
```
Comment (satyam73) [inline events.html:148]: There are mulitple issues mentioned in the ticket, but there's very small change here.
Not getting this thing🤔

**Review by satyam73 — CHANGES_REQUESTED** [review]

**Review by pallabez — APPROVED** [review]
LGTM

**Review by satyam73 — APPROVED** [review]
LGTM

File: events.html (hunk 1 above)
Comment (satyam73) [reply events.html:148]: Got it @samyakshah3008

Comment (rohan09-raj) [inline events.html:148]: I think only `Past Events` will be better here, what do you think on this ?

Comment (samyakshah3008) [reply events.html]: Sure, changed.

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**